core/Tracker/TableLogAction/Cache.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Tracker\TableLogAction;
+
+use Piwik\Common;
+use Piwik\Config;
+use Piwik\Container\StaticContainer;
+use Psr\Log\LoggerInterface;
+
+class Cache
+{
+    /**
+     * @var bool
+     */
+    public $isEnabled;
+
+    /**
+     * @var int cache lifetime in seconds
+     */
+    protected $lifetime;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * for tests
+     *
+     * @var int
+     */
+    static public $hits = 0;
```

Comment:
Instead of hack like this, use a mock in tests. If this class is stored in DI, you can inject the mock in integration/system tests.